app/src/main/res/layout/recently_played_story_card.xml

```diff
@@ -8,7 +8,7 @@
 
     <variable
       name="viewModel"
-      type="org.oppia.android.app.home.recentlyplayed.OngoingStoryViewModel" />
+      type="org.oppia.android.app.home.recentlyplayed.PromotedStoryViewModel" />
   </data>
 
   <com.google.android.material.card.MaterialCardView
```

Comment:
@adhiamboperes Please change the id name of `ongoing_story_card_view ` to `promoted_story_card_view `

Reply:
Also in line `31` we are using `clickOnOngoingStoryTile` which should also be renamed to `clickOnPromotedStoryTile`

Reply:
Thanks @rt4914, I have updated these, PTAL.

Reply:
Thanks @rt4914, I have updated these.